### DISCUSSION TITLE
Bug:- Problem with emoji bar when you react to any reply/post in discussions

### DISCUSSION THREAD

**Discussion by aayushchugh** [community/community#27420] [Discussions] [answered]
As you can see in the picture that my emoji bar is out of window and I am not able to click 👍emoji

![image](https://user-images.githubusercontent.com/69336518/181431213-4214f1e5-1e40-4982-a4aa-c6fcac823c65.png)

**Comment by Gisgar3** [comment]
What operating system and browser do you use? This could help the GitHub team come up with a solution to this issue!

**Reply by airtower-luna** [reply]
I see the same with Firefox 103 on Linux.

**Reply by aayushchugh** [reply]
I am using windows 10 with latest Microsoft edge
this problem occurred today only. it was working fine yesterday

**Reply by edyrkaj** [reply]
Hi, 
it might be related to the font changes that you have on your computer.

Try to change your default font configuration and see if it works:

https://support.microsoft.com/en-us/office/add-a-font-b7c5f17c-4426-4b53-967f-455339c564c1

Thnx

**Reply by Gisgar3** [reply]
Weirdly enough, this issue just started appearing on my end too; Windows 10 19044.1826 w/ FireFox Dev Edition 103.0b9

**Reply by LumitoLuma** [reply]
Me too on Samsung Internet Browser 17.0.9.34. Filed a bug report in GitHub Support (#1723054).

**Reply by radar-blip** [reply]
Same here. Linux with Firefox

**Comment by ettaboyle** [comment, marked as answer]
👋🏻 Hi folks! Thanks for flagging this. I have added the "bug" label and sent a notification to the Discussions team

**Reply by radar-blip** [reply]
👍🏼

**Reply by ettaboyle** [reply]
Update: Team notified and actively working on it 🚀 ❤️‍🩹

**Reply by radar-blip** [reply]
Awesome. Great work! 💯

**Reply by Gisgar3** [reply]
Good to hear!

**Comment by radar-blip** [comment]
A couple pixels off each side of each emoji might work. Or get rid of one of the emojis. But which one to get rid of? 
Or maybe verticalize the emoji bar, or table-ize it 😄

**Comment by jfullstackdev** [comment]
if I split the screen on Chrome, the same thing happened, 

![image](https://user-images.githubusercontent.com/47092464/181684282-1556eeaa-7b9e-4961-aa73-3d7297b53c57.png)

**Reply by ettaboyle** [reply]
Thanks for the screenshots @jdevstatic 🙌🏻  The engineering team is aware of the bug and actively looking into it.

-- Also, so great to see you picking right up where you left off helping all the people like in the Discourse forum! 💟  Thank you!!

**Reply by jfullstackdev** [reply]
also another bug I see is this, when the Title have emojis, 

it's displaying as text at the preview and when scrolling down, like this 

![image](https://user-images.githubusercontent.com/47092464/181800721-5b7f9935-d0f5-479b-a3e7-1d530e8d8fb0.png)

another one, which is not a bug but more of readability, is when the thread is too long, it will have a `View more`, which new messages are hard to see, like this

![image](https://user-images.githubusercontent.com/47092464/181802144-950f2763-74a3-4317-a820-0228fe02e921.png)

my suggestion is to load all but the lazy loading should happen at the bottom when scrolling down but not in the middle

thanks too !
